.shippable.yml

```diff
@@ -0,0 +1,18 @@
+build_image: freedomjs/freedom
+language: node_js
+node_js:
+- '0.10'
+notifications:
+  email:
+    recipients:
+      - freedom@cs.washington.edu
+    on_success: change
+    on_failure: always
+before_install:
+  - "export DISPLAY=:10.0"
+  - "Xvfb :10 -screen 0 1280x1024x24 &"
+install:
+  - "git pull"
```

Comment:
doesn't the default shippable image come with chrome? or even if it doesn't you can install it as a preinstall script for the default image instead of in the docker file. their caching seems reasonable and that process takes <1 min on grunt-jasmine-chromeapp